core-services/template-deprecation/_allowlist.yaml

```diff
@@ -1888,6 +1888,9 @@ templates:
           release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.6:
             generated: false
             kind: release
+          release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.7:
+            generated: false
+            kind: release          
```

Comment:
You have a white space issue on this line :)
Check if you have spaces / tabs after release